app/views/spina/admin/projects/index.html.haml

```diff
@@ -17,7 +17,7 @@
 .well
   - if @projects.any?
     .dd#pages_list
-      %ol.dd-list
+      %ol.dd-list.sortable
```

Comment:
```suggestion
      %ol.dd-list#pages_list{data: {:"sort-url" => spina.sort_admin_products_url }}
```

Reply:
This is why I haven't used what Spina uses. It creates nested projects which might not causes any rendering issues (untested) but it is bad UX.

<img width="1132" alt="Screenshot 2019-10-31 at 18 57 52" src="https://user-images.githubusercontent.com/50486078/67977636-6410a700-fc10-11e9-9f40-2c849b2008d2.png">